zerver/views/message_send.py

```diff
@@ -210,12 +210,14 @@ def send_message_backend(
     tz_guess: Optional[str] = REQ("tz_guess", default=None, documentation_pending=True),
     time: Optional[float] = REQ(default=None, converter=to_float, documentation_pending=True),
 ) -> HttpResponse:
+    recipient_type_name = message_type_name
```

Comment:
This can be done a bit more cleanly be using the first positional argument to `REQ`, to not have set `message_type_name` at all.

Reply:
Ahh, but it gets replaced in the next commit anyway, yeah this is pretty reasonable as an intermediate state before we redo the internals plumbing to support `"direct"` rather than `"private"`.